### DISCUSSION TITLE
More theme / color / appearance

### DISCUSSION THREAD

**Discussion by TBDwarf** [tutao/tutanota#5697] [General]
In my opinion, Tutanota's color choices are lacking today! 
Customization of your e-mail client is certainly something basic, but at the same time so essential. 
One of Proton Mail's strengths today is certainly to offer greater freedom of color/theme choice.

Without going into in-depth modification, a simple increase in choice possibilities would be a real plus! A few more color ranges for the light and dark themes (I assume 90% of the work relies on a little CSS).
Typically, my wife doesn't want to switch to Tutanota, because she uses a dark mode on her phone and can't stand Tutanota's black/green mix (which reminds her of an old MSDOS-style terminal...).
For her, aesthetics can't be sacrificed for privacy...

**Comment by charlag** [comment]
hi, thank you, we will discuss this!

we *do* allow more customization but currently it is gated behind the "whitelabel" feature

**Reply by TBDwarf** [reply]
Thank you very much for your feedback.
I can understand this paying feature for a company wishing to add its own logo, colors, etc., but for personal use (I think, the majority of users) more customization possibilities would be, I think, appreciated (and a real promotional asset for Tutanota).

**Comment by svwezel** [comment]
Allowing user themes (in a similar way to for example vs-code) would be a nice to have!

**Comment by anderlli0053** [comment]
It would be nice to add a theme that is in-between the dark one and light one, because the dark is too dark and the light one is too bright for specific environments. Maybe something similar to what https://catppuccin.com/palette/ offers as "Macchiato" or "Frappé"..?
